notification/events.go

```diff
@@ -338,13 +338,13 @@ func isHealthReportable(events []string, previousHealth, currentHealth models.He
 	return healthDegraded
 }
 
-// getEnvForEvent gets the environment variables for the given event
+// GetEnvForEvent gets the environment variables for the given event
 // that'll be passed to the cel expression or to the template renderer as a view.
-func getEnvForEvent(ctx context.Context, event models.Event, properties map[string]string) (*celVariables, error) {
```

Comment:
All the properties that we need are in the event itself. The 3rd param isn't required.